bundles/org.openhab.binding.avmfritz/src/main/feature/feature.xml

```diff
@@ -6,9 +6,8 @@
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-upnp</feature>
         <bundle dependency="true">mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.activation-api-1.1/2.9.0</bundle>
-        <bundle dependency="true">mvn:javax.xml.bind/jaxb-api/2.2.12</bundle>
-        <bundle dependency="true">mvn:com.sun.xml.bind/jaxb-impl/2.2.11</bundle>
-        <bundle dependency="true">mvn:com.sun.xml.bind/jaxb-core/2.2.11</bundle>
+        <bundle dependency="true">mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.jaxb-api-2.2/2.9.0</bundle>
```

Comment:
There had been features created for such stuff, so no need to depend on bundles (use the feature):
https://github.com/openhab/openhab-core/pull/920/files#diff-ed272408a699f2280d7706c517dc3c53R111-R122

Reply:
:facepalm: Yes, of course. I will change it here and for some other bindings too.

Reply:
Done. See #5878.